tesler-doc-db/src/main/resources/db/migration/liquibase/data/latest/widgets/navigation/navigation-tabs.md

```diff
@@ -1,2 +1,72 @@
 # Tabs
+Tabs are needed to navigate at screen level between different views. They have their own hierarchy: first-level menu tabs are seen always on the current screen, second-level tabs are seen when you choose a tab from first-level menu, etc.
 
+For example, on the current page you can see first-level menu (*"Navigation"*, *"Widgets"*, *"Fields"*) and current second-level menu (*"Overview"*, *"Screen"*, ...). If you choose another first-level tab, you will see another set of second-level tabs. 
+
+More info about level-menu hierarchy you can find at *[Major Concepts](#/screen/getting-started/view/screen/)*
+
+## How to create?
+Tabs are set in **screen.json** files. First-level tabs are in **menu** block. It can have only a view or some child-views. For example:
+```ts
+{
+  "name": "example",
+  "title": "Example Title",
+  "primaryViewName": "no-child-view",
+  "primaryViews": [
```

Comment:
Not required and unused field 
https://github.com/tesler-platform/tesler-schema/blob/master/src/files/ScreenMeta.ts#L43